util/unifycr/src/unifycr-rm.c

```diff
@@ -293,19 +293,37 @@ int unifycr_read_resource(unifycr_resource_t *resource)
  * $ mpirun -n NUMNODES -np NUMNODES unifycrd
  */
 static char *mpirun_newargv[] = {
-    "mpirun", "-n", NULL, "-np", NULL, NULL, (char *) NULL
+    "mpirun", "-n", NULL, "-ppn", "1", "-hosts", NULL, NULL, (char *) NULL
 };
```

Comment:
@sandrain When I run I get:

```bash
catalyst321{sikich1}41: build/util/unifycr/src/unifycr start
mpirun: Error: unknown option "-ppn"
Type 'mpirun --help' for usage.
```
On openmpi/2.0.0

Reply:
@sandrain What mpi/version are you using? @adammoody also suggested maybe trying "mpiexec" here? I think it should have more standardized options across mpi's/versions.

Reply:
It's mpich-3.2, from redhat rpm (mpich-3.2-3.2-2.el7.x86_64). May I ask what's there in catalyst?

Reply:
@sandrain I was using mvapich2/2.2, and I also tried mvapich2/2.3. When using mvapich here you can only use srun. You can only use mpirun here after loading openmpi.